content/distributed.md

```diff
@@ -73,22 +73,33 @@ Please step out of the repository and check that you fork the **forking**-workfl
 
 ## Exercise preparation
 
-```{prereq} Exercise preparation
-**Helpers in breakout-rooms**:
-- Create an exercise repository by
-  [generating from a template](https://help.github.com/en/articles/creating-a-repository-from-a-template)
-  using this template: <https://github.com/coderefinery/template-forking-workflow-exercise>
-  called `forking-workflow-exercise`
-- In this case we **do not add collaborators** to the repository (this is the point of this example).
-- Share the link to the newly created repository in the shared document with your group.
-
-**Learners in breakout-rooms**: Fork the helper's newly created repository and clone the fork.
-
-**Instructor**: Prepare an exercise repository for participants following via stream (see below).
-
-**Learners following via stream**: Fork the repository created by the
-instructor on stream and then clone the fork to your computer.
-```
+``````{prereq} Exercise preparation
+`````{tabs}
+  ````{tab} Part of team/ exercise room
+  **Exercise leads**:
+  - Create an exercise repository by
+    [generating from a template](https://help.github.com/en/articles/creating-a-repository-from-a-template)
+    using this template: <https://github.com/coderefinery/template-forking-workflow-exercise>
+    called `forking-workflow-exercise`
+  - In this case we **do not add collaborators** to the repository (this is the point of this example).
+  - Share the link to the newly created repository in the shared document with your group.
+  
+  **Learners in team/breakout-room**: Fork the helper's newly created repository and clone the fork.
+  ````
+
+  ````{tab} Participating via stream
+  We create(d) these the day before hopefully:
+  - <https://github.com/cr-workshop-exercises/forking-workflow-exercise> (this will not be shown on stream)
```

Comment:
```suggestion
  - <https://github.com/cr-workshop-exercises/forking-workflow-exercise> (this will not be shown on stream or recorded in our videos, but is be public on the Internet until it is deleted)
```